editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -97,6 +105,8 @@ bool TileSetAtlasSourceEditor::TileSetAtlasSourceProxyObject::_get(const StringN
 }
 
 void TileSetAtlasSourceEditor::TileSetAtlasSourceProxyObject::_get_property_list(List<PropertyInfo> *p_list) const {
+	p_list->push_back(PropertyInfo(Variant::NIL, TTR("Atlas"), PROPERTY_HINT_NONE, String(), PROPERTY_USAGE_CATEGORY));
+	p_list->push_back(PropertyInfo(Variant::INT, "id", PROPERTY_HINT_RANGE, "0," + itos(INT_MAX) + ",1"));
```

Comment:
There's not reason to move the ID property here I believe, no ? Like, you could simply use `ADD_PROPERTY(PropertyInfo(Variant::NIL, TTR("Atlas"), PROPERTY_HINT_NONE, String(), PROPERTY_USAGE_CATEGORY))` just before `ADD_PROPERTY(PropertyInfo(Variant::INT, "id", PROPERTY_HINT_RANGE, "0," + itos(INT_MAX) + ",1"), "set_id", "get_id");` instead.

That would avoid some unnecessary changes IMO.

Reply:
Properties added with `ADD_PROPERTY` comes before ones defined here. I moved ID here so it's after the category entry.

Reply:
Well, yeah, but the category can also be defined with `ADD_PROPERTY`. So there's no need to move it here, you can simply move category creation to the bind_method function.

Reply:
`ADD_PROPERTY` needs a getter and setter, so a category can't be created with it. ClassDB might allow using empty string as getter/setter, but it feels hacky and there are no such usage in the codebase.

Reply:
Ah ok, yeah that explains it then. Ok!